tests/test_crawler.py

```diff
@@ -205,7 +205,7 @@ def create_crawler(node_db_filepath: str = IN_MEMORY_FILEPATH, dont_set_teacher:
     teacher_nodes = None
     if not dont_set_teacher:
         teacher_nodes = actions.load_seednodes(None,  # TODO: Needs emitter
-                                               teacher_uris=['https://discover.nucypher.network:9151'],  # TODO: Needs Cleanup
+                                               teacher_uris=['https://gemini.nucypher.network:9151'],  # TODO: Needs Cleanup
```

Comment:
Does it make sense to reference `DEFAULT_TEACHER` here?

Reply:
Sure. Note however that this code doesn't get used in testing due to other blockchain testing issues.